test/Common.FunctionalTests/Inprocess/LoggingTests.cs

```diff
@@ -91,6 +81,41 @@ public class LoggingTests : IISFunctionalTestBase
             }
         }
 
+        [ConditionalTheory]
+        [InlineData("CheckErrLogFile", true)]
+        [InlineData("CheckLogFile", true)]
+        [InlineData("CheckErrLogFile", false)]
+        [InlineData("CheckLogFile", false)]
+        public async Task CheckStdoutLoggingToPipeWithFirstWrite(string path, bool firstWrite)
+        {
+            var deploymentParameters = Helpers.GetBaseDeploymentParameters(publish: true);
+
+            var deploymentResult = await DeployAsync(deploymentParameters);
+            var firstWriteString = path + path;
+
+            if (firstWrite)
+            {
+                Helpers.ModifyEnvironmentVariableCollectionInWebConfig(deploymentResult, "ASPNETCORE_INPROCESS_INITIAL_WRITE", firstWriteString);
+            }
+
+            await Helpers.AssertStarts(deploymentResult, path);
+
+            StopServer();
+
+            if (deploymentParameters.ServerType == ServerType.IIS)
+            {
+                Assert.DoesNotContain(TestSink.Writes, context => context.Message.Contains("TEST MESSAGE"));
```

Comment:
What's the point of this verification? This test doesn't seem to assert anything useful on full IIS